orderbook/src/api/get_markets.rs

```diff
@@ -75,20 +75,14 @@ fn get_amount_estimate_response(
 ) -> impl Reply {
     match result {
         Ok(price) => {
-            let (amount, token) = match query.kind {
-                OrderKind::Buy => (
-                    query.amount.to_big_rational() * price,
-                    query.market.base_token,
-                ),
-                OrderKind::Sell => (
-                    query.amount.to_big_rational() / price,
-                    query.market.quote_token,
-                ),
+            let amount = match query.kind {
+                OrderKind::Buy => query.amount.to_big_rational() * price,
+                OrderKind::Sell => query.amount.to_big_rational() / price,
```

Comment:
nitpick: refactor to a `let amount = query.amount.to_big_rational()` to simplify both branches